### ISSUE TITLE
Helmet Detection using DL

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#218]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Helmet Detection</br>
:red_circle: **Aim** : Create a DL model which will identify the helmets while the workers are doing their jobs.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/whenamancodes/helmet-detection-at-work-for-safety</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.
**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Manoj-2702** [comment]
Full name : Manoj Kumar H S
GitHub Profile Link : https://github.com/Manoj-2702
Email ID : hsmanojkumar2003@gmail.com
Approach for this Project : As I have worked on Object Detection Models before, I have a slight hand of this CNN architectures. 
What is your participant role? Contributor, SSOC-Season-2

So please assign me this role!!
Thank you

**Comment by Rahul-Talari** [comment]
Full name : Talari Rahul Mani Datta
GitHub Profile Link :https://github.com/Rahul-Talari/
Email ID : [20131a1252@gvpce.ac.in](mailto:20131a1252@gvpce.ac.in)
Participant ID (if applicable):
Approach for this Project :
1.Algorithm Selection:
•Choose VGG16, ResNet50, and MobileNetV2 as the deep learning algorithms for the task of classifying children vs. adults.
2.Dataset Split:
• Split the dataset into training and testing sets, with approximately 70% for training and 30% for testing.
•Shuffle the dataset before splitting to ensure a representative distribution of samples in both sets.
3.Implementation:
• Use TensorFlow or PyTorch as the deep learning framework.
• Implement VGG16, ResNet50, and MobileNetV2 architectures.
• Configure the models with appropriate settings and adjustments.
4.Training and Validation:
• Compile each model with a suitable loss function.

What is your partici
pant role? SSOC Participant

I hope that i can successfully manage this and i also know intel api's to peform better optimization so ,I think ,
Ican do justification.

**Comment by abhisheks008** [comment]
Hi all, after evaluating the approaches of both the contributors, i have decided that, @Rahul-Talari will be assigned to this issue once the program starts officially ie. June 1.

Till then, explore more in open source.

@Manoj-2702 you can take up other issues from the Issues section.

**Comment by Rahul-Talari** [comment]
Thank you sir

**Comment by HarshaliD** [comment]
Full name : Harshali Devi
GitHub Profile Link : https://github.com/HarshaliD
Email ID : harshalidevi@gmail.com
Approach for this Project : 
1)Data preprocessing will be the initial step.
2)Various CNN models such as ResNet, DenseNet, and MobileNetV2 will be explored.
3)TensorFlow and Keras will be used for implementing the models.
What is your participant role? : GSSOC 24

**Comment by abhisheks008** [comment]
Nice approach. Issue assigned to you @HarshaliD
